configure

```diff
@@ -3115,6 +3129,22 @@ case $_backend in
 		;;
 esac
 
+#
+# In raspberry Pi, we don't use find_sdlconfig since we could be crosscompiling, but still we use SDL
+#
```

Comment:
I'd argue that in a properly setup cross compiling environment, the configure script should actually be able to use `sdl-config` -- namely, it should "see" the sdl-config for the cross compiling SDL, and get the right linker flags from it.

I think that's what several existing backends do, e.g. I assume (but do not know for sure) that the OpenPandora and SamsungTV backends, which do use find_sdlconfig, are cross-compiled.

But if that is impossible for you for some reason, you should at least imitate the other backends, and setup the require DEFINES and `add_line_to_config_mk`  the way they do it -- in this case, webos and wince (search for `SDL_BACKEND` in this file to see what I mean)